components/zoom/actions/list-call-recordings/list-call-recordings.mjs

```
@@ -0,0 +1,67 @@
+      min: 1,
+      max: 300,
+    },
+  },
+  async run ({ $ }) {
+    const {
+      zoom, startDate, endDate, max,
+    } = this;
+    let to = endDate;
+    if (startDate && !endDate) {
+      const date = new Date(startDate);
+      if (isNaN(date.valueOf())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)